assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
+                "OMN": ["OMN", "PCMAN", "MLB", "TRV", "PHORD", "PBI"],
+                "SNSBK": ["SNSBK"],
+                "TRV": ["TRV", "PHORD", "PBI"]
+            },
+            "body": ["ANNEY", "HILEY", "KAINS", "VKZ"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["OMN*", "PCMAN", "MLB*", "TRV*", "PHORD", "PBI", "ANNEY", "HILEY", "KAINS", "VKZ"],
+                ["SNSBK*", "ANNEY"]
+            ]
+        },
+        "BLUFI4": {
+            "icao": "BLUFI4",
+            "name": "Bluffy Four",
+            "entryPoints": {
+                "MLB": []
+            },
+            "body": ["MLB", "LEBUR", "DALOO", "TOPPR", "BLUFI", "HEATT", "KAINS", "VKZ"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
             },
-            "body": ["HILEY", "KAINS", "VKZ"],
-            "draw": [["OMN*","PCMAN","MLB","TRV","PHORD","PBI","ANNEY","HILEY"], ["ZFP*","SNSBK"], ["HILEY","KAINS","VKZ"]]
+            "draw": [
+                ["MLB*", "LEBUR", "DALOO", "TOPPR", "BLUFI", "HEATT", "KAINS", "VKZ"]
+            ]
         },
         "CURSO4": {
             "icao": "CURSO4",
             "name": "Curso Four",
             "entryPoints": {
-                "EADEN": ["EADEN", "STRAP", "TIGAR", "CARNU", ["DVALL", "A120-"]],
-                "MTH": [["MTH", "S280"], "GUSHY", ["DVALL", "A120-"]]
+                "EADEN": ["EADEN", "STRAP", "TIGAR", "CARNU"],
+                "MTH": ["MTH", "GUSHY"]
+            },
+            "body": ["DVALL", "CURSO", "MNNDY", "JODPO"],
+            "rwy": {
+                "KMIA8L": ["BEETR", "#053"],
+                "KMIA8R": ["BEETR", "#053"],
+                "KMIA9": ["BEETR", "#053"],
+                "KMIA12": ["BEETR", "#053"],
+                "KMIA26L": ["LECIT", "#124"],
+                "KMIA26R": ["LECIT", "#124"],
+                "KMIA27": ["LECIT", "#124"],
+                "KMIA30": ["LECIT", "#124"]
             },
-            "body": [["CURSO", "A100|S250"], "MNNDY", "JODPO", "BEETR"],
-            "draw": [["EADEN*","STRAP","TIGAR","CARNU","DVALL"],["MTH*","GUSHY","DVALL"],["DVALL","CURSO","MNNDY","JODPO","BEETR"]]
+            "draw": [
+                ["EADEN*", "STRAP", "TIGAR", "CARNU", "DVALL", "CURSO", "MNNDY", "JODPO", "BEETR"],
+                ["MTH*", "GUSHY", "DVALL"],
+                ["JODPO", "LECIT"]
+            ]
         },
-        "CYY7": {
-            "icao": "CYY7",
-            "name": "Cypress Seven",
+        "CYY8": {
```

Comment:
CYY8 is now CYY9

Reply:
reverified and updated all SIDs/STARs.

done